tasks/post_copy_request_to_queue/post_copy_request_to_queue.py

```diff
@@ -140,9 +139,9 @@ def task(
             my_base_delay = exponential_delay(my_base_delay, retry_backoff)
             continue
     else:
-        message = f"Error sending message to {recovery_queue_url} for {new_data}"
-        logging.critical(message)
-        raise Exception(message)
+        message = "Error sending message to recovery_queue_url for {new_data}"
+        LOGGER.critical(message, new_data = str(new_data))  #PO DAAC needs to fix cumulus logger to handle dict
```

Comment:
Change comment from `#PO DAAC needs to fix cumulus logger to handle dict` to `#Cumulus will update this library in the future to be better behaved.` or something similar.